Bottom: 13b86d0e7b9b6f9c44b40d9b12c409a8d77545d1
Top:    949cb33ec34dd47c03e03b463c796377f1c514e9
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-06 16:06:08 +0000

Refresh of fix-poly-contours

---

diff --git a/src/polygon.c b/src/polygon.c
index 6824d45..b0cc0e0 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -726,9 +726,6 @@ LinePoly (LineType * L, Coord thick)
   if ((contour = poly_NewContour (poly_CreateNode (v))) == NULL)
     return 0;
 
-  v[0] = l->Point2.X - dx;
-  v[1] = l->Point2.Y - dy;
-
   if (TEST_FLAG (SQUAREFLAG,l))
     {
       v[0] = l->Point2.X + dx;
